CesiumIonClient/src/Connection.cpp

```diff
@@ -582,6 +588,85 @@ Defaults defaultsFromJson(const rapidjson::Document& json) {
   return defaults;
 }
 
+GeocoderResult geocoderResultFromJson(const rapidjson::Document& json) {
+  GeocoderResult result;
+
+  auto featuresMemberIt = json.FindMember("features");
+  if (featuresMemberIt != json.MemberEnd() &&
+      featuresMemberIt->value.IsArray()) {
+    auto featuresIt = featuresMemberIt->value.GetArray();
+    for (auto& feature : featuresIt) {
+      const rapidjson::Value* pLabel =
+          rapidjson::Pointer("/properties/label").Get(feature);
+      if (!pLabel) {
+        SPDLOG_WARN("Missing label for geocoder feature");
+        continue;
+      }
+
+      std::string label(pLabel->GetString());
```

Comment:
What if `properties.label` isn't a string? It'd be safer to either check `IsString`, or use `JsonHelpers::getStringOrDefault` (which will do this for you).